.github/workflows/main.yml

```diff
@@ -5,10 +5,11 @@ on:
     branches:
       - main
   pull_request:
-    types: [opened, synchronize, reopened]
+    types: [opened, synchronize, reopened, labeled]
 
 jobs:
   build:
+    if: github.ref == 'refs/heads/main' || contains(github.event.pull_request.labels.*.name, 'tagged')
```

Comment:
This will prevent the build from running unless the pr is tagged
We currently use these builds as required checks
Maybe just apply the logic to the publish step?

Reply:
As per our conversation - we are going to remove this as the GitHub doesn't have a limit on build minutes on public repos 🚀